### ISSUE TITLE
Continuation DID Document URL should support Hashlink

### ISSUE THREAD

**Issue by peacekeeper** [w3c-ccg/didm-btcr#14]
The spec says that if the Continuation DID Document is in a mutable store, it must be signed with the transaction signing key along with a timestamp. But the problem with this is that the document could change at any time. Instead we can support Hashlink which solves this and has other benefits as well:

https://github.com/w3c-ccg/community/issues/48

**Comment by kimdhamilton** [comment]
Yes this is a great suggestion. We didn’t have hashlink in our toolset at the time, but it will greatly cleanup the awkwardness in the BTCR method spec around mutable/immutable store handling
